tests/global_config_arm64.py

```diff
@@ -67,6 +67,10 @@
     os_name=OS_FLAVOR_FEDORA, preferences=[OS_FLAVOR_FEDORA], arch_suffix=ARM_64
 )
 
+(
```

Comment:
to keep the convention between global_config files, please move this assignment after `fedora_os_matrix` - https://github.com/RedHatQE/openshift-virtualization-tests/pull/2321/files#diff-01ab1d2f2a8c8c2dd3e7254d02f0609fbb508443d2674885257ed6d59c3e6cdcR57